src/hacbs/components/ApplicationDetails/__tests__/HacbsApplicationDetails.spec.tsx

```diff
@@ -14,21 +15,66 @@ jest.mock('@openshift/dynamic-plugin-sdk-utils', () => ({
   useK8sWatchResource: jest.fn(),
 }));
 
+jest.mock('@redhat-cloud-services/frontend-components/useChrome', () => ({
+  useChrome: jest.fn(),
+}));
+
+const useChromeMock = useChrome as jest.Mock;
+
 configure({ testIdAttribute: 'data-test' });
 
 const watchResourceMock = useK8sWatchResource as jest.Mock;
 
-describe('HacbApplicationDetails', () => {
+describe('HacbsApplicationDetails', () => {
   it('should render spinner if application data is not loaded', () => {
     watchResourceMock.mockReturnValue([[], false]);
-    render(<HacbApplicationDetails applicationName="test" />);
+    const set = jest.fn();
+    const toggle = jest.fn();
+    useChromeMock.mockReturnValue({
+      quickStarts: { set, toggle },
+    });
+    render(<HacbsApplicationDetails applicationName="test" />);
     expect(screen.queryByTestId('spinner')).toBeInTheDocument();
   });
 
   it('should render application display name if application data is loaded', () => {
     watchResourceMock.mockReturnValueOnce([mockApplication, true]);
-    render(<HacbApplicationDetails applicationName="test" />);
+    const set = jest.fn();
+    const toggle = jest.fn();
+    useChromeMock.mockReturnValue({
+      quickStarts: { set, toggle },
+    });
+    render(<HacbsApplicationDetails applicationName="test" />);
     expect(screen.queryByTestId('details__title')).toBeInTheDocument();
     expect(screen.queryByTestId('details__title').innerHTML).toBe('Test Application');
   });
+
+  it('should only display quick start on first run', () => {
+    const showQuickStart = localStorage.getItem('hacbs/showApplicationQuickstart');
+    localStorage.removeItem('hacbs/showApplicationQuickstart');
+    expect(showQuickStart).toBeNull;
+    watchResourceMock.mockReturnValueOnce([mockApplication, true]);
+    const set = jest.fn();
+    const toggle = jest.fn();
+    useChromeMock.mockReturnValue({
+      quickStarts: { set, toggle },
+    });
+
+    render(<HacbsApplicationDetails applicationName="test" />);
+    // localStorage key doesn't exist, quickstart set/toggle should be called
+    expect(showQuickStart).toEqual('false');
```

Comment:
You haven't fetched the latest value from `localStorage` therefore this assertion is invalid.

Reply:
very true, good catch. this is all fixed now.